### ISSUE TITLE
Support for deploying to a website path instead of a website root

### ISSUE THREAD

**Issue by ciwchris** [Doctave/doctave#10]
It appears the generated site can only be deployed to the root of a website, ex: "https://example.com/index.html", because leading slashes are used on all assets. What do you think of supporting generating the site so it can be deployed to a path, ex: "https://example.com/documenation/index.html"?

**Comment by begleynk** [comment]
You're correct - this isn't currently supported but sounds like a positive feature to add. 

Is this blocking you from using Doctave? I'd be happy to add this into the next 0.3.0 release with a couple other things but they are still in the works.

**Comment by ciwchris** [comment]
Currently I'm experimenting. I intend to present it to my team next week. Typically it's a slow process though. What will probably end up happening is that a trial go ahead will be given and I'll start to mirror some of our docs. There are other options I can use to deploy to that will work at the root, but deploying to a path will be much easier.

But relatedly, I also tried taking a look at the Doctave service. Unfortunately I ran into a problem with the cert at docs.doctave.com. My company terminates TLS connections at the firewall and inserts their own cert. For docs.doctave.com the browser responds with ERR_HTTP2_INADEQUATE_TRANSPORT_SECURITY. So there's something about the cert my company doesn't like. Not your problem but currently that's going to be a pretty hard blocker for me.
